apps/frontend/app/(client)/(main)/contest/[contestId]/@tabs/page.tsx

```diff
@@ -107,10 +123,11 @@ export default async function ContestTop({ params }: ContestTopProps) {
   const benefits = data.benefits
   const description = data.description
   const prev = true
+  const currentContestId = data.id
```

Comment:
이부분 이렇게 바꾸는게 나을 것 같아요!
```suggestion
    const {
    posterUrl,
    participationTarget,
    competitionMethod,
    rankingMethod,
    problemFormat,
    benefits,
    description,
    id: currentContestId
  } = data

  const imageUrl = posterUrl || '/logos/welcome.png'
```